src/main.ts

```diff
@@ -50,10 +50,11 @@ export class DreamsSDK {
   }
 
   /**
-   * @param token  jwk token for the user
+   * @param token jwk token for the user
    * @param locale determines the localisation configuration that will be applied.
+   * @param location path to which the user will be redirected to after the token is verified
    */
-  start(token: string, locale: string) {
+  start(token: string, locale: string, location?: string) {
```

Comment:
IMO you can disregard this message (or try to improve the code if it'sworth it)

Reply:
Which message do you mean?

Reply:
that's the code climate complaint about code complexity